src/Data/Mapping/Masa.Contrib.Data.Mapping.Mapster/DefaultMappingConfigProvider.cs

```diff
@@ -0,0 +1,192 @@
+﻿// Copyright (c) MASA Stack All rights reserved.
+// Licensed under the MIT License. See LICENSE.txt in the project root for license information.
+
+namespace Masa.Contrib.Data.Mapping.Mapster;
+
+public class DefaultMappingConfigProvider : IMappingConfigProvider
+{
+    private readonly ConcurrentDictionary<(Type SourceType, Type DestinationType, MapOptions? mapOptions), TypeAdapterConfig?>
```

Comment:
Why is SourceType pascal, and mapOptions camel?

Reply:
processed